components/NurseProfile/NurseProfile.styled.js

```diff
@@ -22,59 +22,100 @@ export const StyledNurseProfile = styled.div`
 `;
 
 export const DeleteButtonStyled = styled.button`
-background-color: #fafafc;
-color: #000;
-padding: 0.8rem 1.2rem;
-margin: 5px 15px 5px 30px;
-width: 100px;
-border: none;
-border-radius: 5px;
-text-decoration:none;
-cursor: pointer;
-font-size: 16px;
-transition: 0.3s;
-align-items: center;
-justify-content:center;
+  background-color: #fafafc;
+  color: #000;
+  padding: 0.8rem 1.2rem;
+  margin: 5px 15px 5px 30px;
+  width: 100px;
+  border: none;
+  border-radius: 5px;
+  text-decoration: none;
+  cursor: pointer;
+  font-size: 16px;
+  transition: 0.3s;
+  align-items: center;
+  justify-content: center;
   &:hover {
-    background-color: #EF8059;
+    background-color: #ef8059;
   }
-}
 `;
 
-export const GoBackLinkStyled = styled.button`
-background-color: #fafafc;
-color: #000;
-padding: 0.6rem 1.2rem;
-margin: 5px 15px 5px 30px;
-width: 100px;
-border: none;
-text-decoration:none;
-border-radius: 5px;
-cursor: pointer;
-font-size: 16px;
-transition: 0.3s;
-align-items: center;
-justify-content:center;
+export const GoBackButton = styled.button`
+  background-color: #fafafc;
+  color: #000;
+  padding: 0.6rem 1.2rem;
+  margin: 5px 15px 5px 30px;
+  width: 100px;
+  border: none;
+  text-decoration: none;
+  border-radius: 5px;
+  cursor: pointer;
+  font-size: 16px;
+  transition: 0.3s;
+  align-items: center;
+  justify-content: center;
   &:hover {
-    background-color: #6FC6FF;
+    background-color: #6fc6ff;
   }
-}`;
+`;
 
 export const UpdateButtonStyled = styled.button`
-background-color: #fafafc;
-color: #000;
-padding: 0.8rem 1.2rem;
-margin: 5px 15px 5px 30px;
-width: 100px;
-border: none;
-border-radius: 5px;
-cursor: pointer;
-transition: 0.3s;
-text-decoration:none;
-font-size: 16px;
-align-items: center;
-justify-content:center;
+  background-color: #fafafc;
+  color: #000;
+  padding: 0.8rem 1.2rem;
+  margin: 5px 15px 5px 30px;
+  width: 100px;
+  border: none;
+  border-radius: 5px;
+  cursor: pointer;
+  transition: 0.3s;
+  text-decoration: none;
+  font-size: 16px;
+  align-items: center;
+  justify-content: center;
   &:hover {
     background-color: #48e68b;
   }
-}`;
+`;
+
+export const ContactsContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: #fcfafb;
+  max-width: 500px;
+  padding: 50px 70px;
+  border-radius: 15px;
+  box-shadow: 0px 4px 15px rgba(0, 0, 0, 0.1);
+
+  h2,
+  h3 {
+    margin-bottom: 20px;
+  }
+
+  p {
+    font-size: 16px;
+    color: #333;
+    margin-bottom: 15px;
+    line-height: 1.5;
+  }
+`;
+
+export const WorkScheduleLink = styled(Link)`
```

Comment:
In general you should think about refactoring your code in a future user story. You can reuse a lot of components if you just use a different `naming approach`. 

You should avoid naming purley styled components after their functionality, e.g. BackButton, WorkScheduleLink, SubmitFeedBackButton.
Because you are then forced to only use these components for this very specific task. If you want a button in a different case that should look almost identical you still need to create a new one, since it would be weird to use the "SubmitFeedBackButton" in as your "Delete this item Button". 

The better approach here is to only describe the `look and feel` of your component, e.g. "FilledButton", OutlinedButton", "TransparentButton", "CircularButton". 
You can then use the "FilledButton" for your submit button AND for the delete button. 

You don't have to change it now but it would improve the code quality a lot.

Reply:
Yes. You're right. I will have more attention and create more general button to be used globally and specify some design details when necessary, locally in components.